packages/core/useElementVisibility/index.test.ts

```diff
@@ -34,6 +35,19 @@ describe('useElementVisibility', () => {
           return { stop }
         }),
       }))
+
+      vi.mock('../watchOnce', () => {
+        return {
+          watchOnce: vi.fn((source, callback) => {
+            // Immediately execute the callback if the source is true
+            if (typeof source === 'object' && source.value === true) {
+              callback(true)
+            }
+          }),
+        }
+      })
```

Comment:
why do we need to mock this? 🤔

Reply:
I mock watchOnce to ensure tests run predictably and synchronously. This allows us to control when the callback is triggered, making our tests more reliable and independent of Vue's reactivity system. By doing so, we can focus on testing the logic of useElementVisibility without external timing issues.